tests/plotting/test_plotting.py

```diff
@@ -50,7 +50,7 @@
 
 # IMAGE warning/error thresholds (assumes using use_vtk)
 IMAGE_REGRESSION_ERROR = 500  # major differences
-IMAGE_REGRESSION_WARNING = 200  # minor differences
+IMAGE_REGRESSION_WARNING = 300  # minor differences
```

Comment:
I trust your judgement on this one ;)

Reply:
This was for a separate issue that I shouldn't have done here.  I'll save this for another PR.